static/src/javascripts-legacy/projects/common/modules/lazyload.js

```diff
@@ -23,35 +22,34 @@ define([
             beforeInsert      - function applied to response html before inserting it into container, optional
             success           - callback function, optional
             error             - callback function, optional
-            always            - callback function, optional
             force             - boolean, default false. Reload an already-populated container
         */
+
         options = merge({
```

Comment:
I don't think the options hash is more that one-level deep; you could use `Object.assign` and at least remove the `merge` module here 😃